content_scripts/hints.js

```diff
@@ -331,6 +331,28 @@ var Hints = (function() {
         document.documentElement.prepend(holder);
     }
 
+    function createHintsForElements(elements, attrs) {
```

Comment:
This function duplicates some code from `createHintsForClick` and `createHintsForTextNode` - maybe common logic should be extracted into a separate function?